README.md

```diff
@@ -43,93 +43,94 @@ pip install dash-tools
 
 ## **Usage Examples**
 
-Below are common usage examples. For a more in-depth tutorial on writing apps for Plotly Dash, see the [Plotly Dash Documentation](https://dash.plotly.com/layout). For information about dash-tools commands, read the [_Commands_](#commands) section.
+### A. Create an App
 
-### **Creating A New Project**
-
-> _For a more in-depth tutorial on using dash-tools, check out the [Create and Upload Your App](/docs/Configuring-Your-App.md) document!_
-
-Creating a new Dash project is very simple. The following command will create a new directory called "MyDashApp":
+1. Create a Dash project in a new directory called "MyDashApp" (using your terminal or command prompt):
 
 ```bash
 dash-tools --init MyDashApp
 ```
 
-Optionally, templates can be used. Check out [Templates](#templates) for more details.
-
-You can see what files are included with your new app:
+2. Open the default app:
 
 ```bash
-cd MyDashApp
+.\MyDashApp\src\app.py
```

Comment:
Same comment as below on line 67, this is windows specific code

Reply:
Done